config.yaml

```
@@ -0,0 +1,31 @@
+baseURL: "http://teleport-cop.github.io/"
```

```suggestion
baseURL: "http://goteleport-cop.github.io/"
```

change URL to one that we can manage